### PR TITLE
struct updates, local key add, create account commands

### PR DESCRIPTION
**Pull request by miguel1770**

### CODE REVIEW

File: cmd/fixture_utils/step_runner.go [hunk 1]
```diff
@@ -78,6 +78,60 @@ func WaitForNextBlockWithErrorCheck(t *testing.T) {
...
+// RunCreateAccount is a function to create account
+func RunCreateAccount(step FixtureStep, t *testing.T) {
+	if step.ParamsRef != "" {
+		caKey := GetAccountKeyFromTempName(step.ParamsRef, t)
+		localKeyResult, err := inttest.AddNewLocalKey(caKey)
+		t.WithFields(testing.Fields{
+			"key":              caKey,
+			"local_key_result": localKeyResult,
+		}).MustNil(err, "error creating local Key")
+		t.Log("localKeyResult, err :=", localKeyResult, err)
```
Comment (antstalepresh) [inline cmd/fixture_utils/step_runner.go]: can we remove this log

File: cmd/fixture_utils/step_runner.go [hunk 2]
```diff
@@ -78,6 +78,60 @@ func WaitForNextBlockWithErrorCheck(t *testing.T) {
...
+		result, logstr, err := inttest.CreateChainAccount(caKey)
+		t.WithFields(testing.Fields{
+			"result": result,
+			"logstr": logstr,
+		}).MustNil(err, "error creating account on chain")
+		WaitForNextBlockWithErrorCheck(t)
+		WaitForNextBlockWithErrorCheck(t)
```
Comment (antstalepresh) [inline cmd/fixture_utils/step_runner.go]: We have a function to wait 2 blocks and it won't be necessary to use this function 2 times.

File: cmd/fixture_utils/step_runner.go [hunk 3]
```diff
@@ -78,6 +78,60 @@ func WaitForNextBlockWithErrorCheck(t *testing.T) {
...
+		if err != nil {
+			TxBroadcastErrorCheck(err, txhash, step, t)
+			return
+		}
+
+		WaitForNextBlockWithErrorCheck(t)
+
+		// txHandleResBytes := GetTxHandleResult(txhash, t)
+		// resp := handlers.GetPylonsResponse{}
+		// err = inttest.GetAminoCdc().UnmarshalJSON(txHandleResBytes, &resp)
```
Comment (antstalepresh) [inline cmd/fixture_utils/step_runner.go]: To check GetPylons status message, need to define this within GetPylons handler.

File: cmd/fixture_utils/step_runner.go (hunk 2 above)
Comment (miguel1770) [reply cmd/fixture_utils/step_runner.go]: It's modified

File: cmd/fixture_utils/step_runner.go (hunk 1 above)
Comment (miguel1770) [reply cmd/fixture_utils/step_runner.go]: It's removed

File: cmd/fixture_utils/step_runner.go (hunk 3 above)
Comment (miguel1770) [reply cmd/fixture_utils/step_runner.go]: it's modified

**Review by antstalepresh — APPROVED** [review]
LGTM

**Pull request merged**